src/app/shared/provider.service.ts

```diff
@@ -16,6 +16,17 @@
 
 import { ExtendedDockstoreTool } from './models/ExtendedDockstoreTool';
 import { ExtendedWorkflow } from './models/ExtendedWorkflow';
+import { IconDefinition } from '@fortawesome/fontawesome-common-types';
+import { faBitbucket, faGithub, faGitlab } from '@fortawesome/free-brands-svg-icons';
+import { faDockstore, faWhale } from './custom-icons';
+
+const GITHUB = 'GitHub';
+
```

Comment:
Real nitpicky, I would probably remove the blank lines in between the const's.

Seeing it the way it is now, I would be tempted to make an enum, but that can be done at some other point.

Reply:
Done